installer/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "terraprisma-installer"
+edition = "2021"
+
+version = "0.1.0"
+authors = ["pbone"]
+description = "Native installer for Terraprisma."
+license-file = "../LICENSE"
```

Comment:
We'll eventually need to figure out versioning, kind of a pain. Do we want to increment it in line with any other projects (or at least keep the major and minor versions consistent?).